bundles/org.eclipse.equinox.simpleconfigurator/src/org/eclipse/equinox/internal/simpleconfigurator/ConfigApplier.java

```diff
@@ -435,49 +416,22 @@ private boolean isFragment(Bundle current) {
 		return (revision != null) && ((revision.getTypes() & BundleRevision.TYPE_FRAGMENT) != 0);
 	}
 
-	private void refreshPackages(Bundle[] bundles, BundleContext context) {
-		if (bundles.length == 0 || packageAdminService == null)
+	private void refreshPackages(Collection<Bundle> bundles) {
+		if (bundles.isEmpty())
 			return;
 
```

Comment:
If you change this, you could also add brace.
```suggestion
		if (bundles.isEmpty()) {
			return;
		}
```